docs/installation/SUMMA_on_OS_X.md

```diff
@@ -60,6 +61,43 @@ In the following I will assume that you don't have a Fortran compiler or NetCDF
     * `sudo port install nco`    : to manipulate NetCDF files, see the [NCO homepage](http://nco.sourceforge.net)
     * `sudo port install cdo`    : to manipulate NetCDF files, see the [CDO homepage](https://code.mpimet.mpg.de/projects/cdo/)
 
- 1. Now obtain the SUMMA source code from the [SUMMA source code repository](https://github.com/NCAR/summa). You may just want to download the latest tagged release. Unless you are planning to contribute to the source code, there is no need to clone or fork the repository.
+## Using Homebrew
 
- 1. Untar or unzip the archive, then go to the `summa/build` directory and follow the instructions in the [SUMMA installation](SUMMA_installation.md) page. If you are using MacPorts, the `FC_ENV` can be set to `gfortran-6-macports`.
+1. Install [Homebrew](https://brew.sh/) by opening the terminal and running the following command:
+       
+       /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+1. Install NetCDF library and the Fortran version of the NetCDF library
+
+       brew install netcdf netcdf-fortran
+2. Install gcc and gfortran
+
+       brew install gcc
+4. Install other NetCD utilities
+
+       brew install nco cdo ncview
+5. Install LAPACK
+       
+       brew install lapack
+
+# Download and compile the source code
+
+* Now obtain the SUMMA source code from the [SUMMA source code repository](https://github.com/NCAR/summa). You may just want to download the latest tagged release. Unless you are planning to contribute to the source code, there is no need to clone or fork the repository.
+
+* Untar or unzip the archive, then go to the `summa/build` directory and follow the instructions in the [SUMMA installation](SUMMA_installation.md) page. 
+
+* If you are using MacPorts, the `FC_ENV` can be set to `gfortran-6-macports`.
+
+* If you are using Homebrew, then use the following entries in Part 0 of Makefile:
+
+Note: change `x.x.x` with the exact version number in both the `INCLUDE` and `LIBRARIES` variables. 
+This can be done by using `ls /opt/homebrew/Cellar/netcdf-fortran/` and then using `tab` button to find the current version installed on your machine. 
```

Comment:
Mentioning the older homebrew default locations here could be good too

Reply:
Unfortunately, I don't know the paths for the older homebrew versions.

Reply:
Sounds good, but I can't confirm this at the moment.